### PR TITLE
refactor: retire legacy ClaimWizard contracts

### PR DESCRIPTION
**Pull request by arbenl**
## Summary

- retires the two production-orphaned legacy ClaimWizard trees and their obsolete unit/E2E contracts
- preserves the current dormant Claim Draft Intake and all canonical claim writer/action/domain/upload/provider surfaces unchanged
- removes dangling legacy Playwright and tenant-host inventory entries and synchronizes only the exact repo-size/lint baselines

Governance: accepted gate `IDA-DG21`, sole slice `IDA-CW01`.

- gate SHA-256: `c84734f482f73682c1f7123ac3cef8e8b58057615ef8d189a4ac7b401e550736`
- runtime authority SHA-256: `44ad2c2d41ab06901369612e854b0310b9de8a33258bd9606575294c22f5eeae`
- exact base/parent: `48c7704b46ace6d1ca8ba489e536ae395df26469`
- exact head: `2030d7904d526948c9d8f004c40714c5697b0ea4`
- exact shape: 25 deletions + 10 modifications + zero new files
- exact diff digest: `f5a93724983aaf53b71c5b65729d51c50a2719f0a67382e0605711cb318fb01d`

## Changed areas

- deleted 12 unimported legacy ClaimWizard production modules
- deleted 6 legacy unit-test files and 7 obsolete/submission-dependent legacy Playwright specs
- updated the current dormant-intake route/unit/E2E assertions
- preserved the state-dependent member hero CTA contract, then reaches intake through visible Claims and create-claim affordances (no direct navigation bypass)
- removed the deleted scenario's named Playwright configuration entry
- synchronized exact tenant-host allowlist/count, lint-warning baseline, task helper, and repo-size budget

No proxy, route, auth, session, shared-auth, tenancy, schema, RLS, database, dependency, workflow, provider, deployment, canonical claim writer/action/domain/upload/event/notification, or successor change is included.

## Exact-head verification

- P00 revalidation: PASS — all 12 per-path importer closures; no dangling named Playwright entry; c1-04/c1-05/scenario-01 preservation; protected inventories byte-identical
- focused KS/SQ + MK/MK real-navigation proof on predecessor `f36d7cd64`: PASS (setup plus both locale/tenant lanes); the accepted incremental head changes only a skipped live legacy spec and deterministic inventories
- `pnpm security:guard`: PASS
- predecessor `f36d7cd64` `pnpm pr:verify:hosts`: PASS
  - CI contracts 353/353
  - coverage 85.60%
  - E2E 209 passed / 9 expected skips
  - smoke 13 passed / 11 expected skips
- predecessor `f36d7cd64` `pnpm e2e:gate`: PASS — 209 passed / 9 expected skips
- exact-head tenant-host contracts: PASS — 5/5 and 12 projects / 12 files
- exact-head Playwright listing: PASS — zero deleted-spec selections
- exact-head deterministic repo-size sync/check: PASS
- Opus architecture/contracts/security review: OVERALL PASS
- Codex Security original current diff scan: COMPLETE — 32/32 source-like paths closed, zero findings
- Codex Security lifecycle-disposition delta scan: COMPLETE — 3/3 source-like paths closed, zero findings; report SHA-256 `92632c29f86fe499e0d5038547a772b23facd103382caff47c2d3fef79681585`

`pnpm slice:verify` passed all slice static/security checks and web unit coverage; its only non-pass was the untouched main baseline database test that still searches the schema barrel for an index now defined in `claim-core.ts`. `packages/database` is unchanged by this PR.

## Edge cases and residual risk

- C1 closed-loop submission coverage is intentionally retired and is not claimed; a future separately governed submission gate must create fresh contracts
- writer-independent tenant isolation, RBAC, staff assignment/status, internal-note, upload, and member-read coverage remains in surviving contracts
- current dormant submit remains visible, disabled, inert, and unable to produce `claim-created-success`

## Rollback / operations

Rollback is a single PR revert restoring the orphaned files and stale contracts. This PR authorizes no deployment. After a green/classified merge, the automatic CD run must be cancelled before registry login, image build, or any deploy job.

### CODE REVIEW

**Comment by arbenl** [comment]
@copilot review

File: apps/web/e2e/ui-v2-onboarding.spec.ts [hunk 1]
```diff
@@ -2,30 +2,58 @@ import { expect, test } from './fixtures/auth.fixture';
...
 
-    await expect(page.locator('[data-ui-v2="true"]')).toBeVisible();
+    const primaryCta = page.getByTestId('member-primary-actions').locator('a');
+    await expect(primaryCta).toBeVisible();
+    const primaryHref = await primaryCta.getAttribute('href');
+    if (!primaryHref) throw new Error('Expected the member dashboard primary CTA to have an href');
+    await primaryCta.click();
+    await expect(page).toHaveURL(new URL(primaryHref, page.url()).pathname);
+
+    await gotoApp(page, routes.memberNewClaim(testInfo), testInfo, {
```
[the remark this answers is not in the document]
Comment (arbenl) [reply apps/web/e2e/ui-v2-onboarding.spec.ts]: Resolved on current head f36d7cd641f6753909aee4b6f5e136172e578a37 without assuming a fixed hero destination. The test now asserts the primary hero CTA navigates to its exact state-dependent visible href (the seeded KS member legitimately lands on its active-claim detail), then uses the visible Claims navigation and visible create-claim-button to reach the dormant intake, awaits new-claim-page-ready, and retains all intake assertions. There is no direct gotoApp bypass. Focused KS/MK navigation proof passed, and exact-head e2e:gate passed 209 tests with 9 expected skips.

**Comment by arbenl** [comment]
@codex review current head f36d7cd641f6753909aee4b6f5e136172e578a37. @copilot review current head f36d7cd641f6753909aee4b6f5e136172e578a37 in review-only mode; do not push commits or modify the branch. The stale P2 is resolved by exact state-dependent CTA navigation plus visible Claims/create-claim navigation, with focused and full E2E proof recorded in the PR body.

File: apps/web/e2e/live/pilot-day1-drive.spec.ts [hunk 2]
```diff
@@ -1,104 +0,0 @@
...
-      await expect(page.locator('h1')).toContainText('New Claim');
-
-      // 3. Step 1: Category
-      await page.getByTestId('category-vehicle').click({ force: true });
-      await page.waitForTimeout(600); // Wait animation
-      await page.getByTestId('wizard-next').click({ force: true });
-
-      // 4. Step 2: Details
-      await page.waitForLoadState('domcontentloaded');
-      const claimTitle = `Live Pilot Driver Claim #${i} - ${new Date().toLocaleTimeString()}`;
```
[the remark this answers is not in the document]
Comment (arbenl) [reply apps/web/e2e/live/pilot-day1-drive.spec.ts:71]: Resolved on exact current head 2030d7904d526948c9d8f004c40714c5697b0ea4 under accepted disposition ef565d2b313d319364ab8a49bcfb02aac29dc99aafedf023e1650cdae2debf2b. The orphaned live lifecycle consumer is deleted; its tenant-host allowlist row is removed; the guard now proves 12 projects / 12 files; and the size budget tightened by the exact 5,306-byte spec plus 85-byte row. No replacement producer, claim writer, DB, runtime, auth, tenant, or provider path was introduced. Focused contracts, Opus architecture/contracts/security review, and the bounded Codex Security 3/3 scan all passed with zero findings.
